toc/working-groups/docs.md

```diff
@@ -60,4 +60,31 @@ areas:
   - cloudfoundry/docs-credhub
   - cloudfoundry/docs-deploying-cf
   - cloudfoundry/docs-dotnet-core-tutorial
+- name: Cloud Foundry Tutorials
+  approvers:
+  - name: Steve Greenberg
+    github: spgreenberg
+  - name: Anita Flegg
+    github: anita-flegg
+  reviewers:
+  - name: Benjamin Guttmann
+    github: benjaminguttmann-avtq
+  - name: Maurice Brinkmann
+    github: mauricebrinkmann
+  - name: Andreas Koppenhöfer
+    github: akop
+  - name: Jovan Kostovski
+    github: chombium
+  - name: Arsalan Khan
+    github: asalan316
+  repositories:
+  - cloudfoundry-tutorials/cf4devs
```

Comment:
Currently you can't have repos from different managed github orgs in one WG. The github org is defined on WG level (default: cloudfoundry). Validation seems to have room for improvement.